src/Fantomas/Trivia.fs

```diff
@@ -145,9 +145,9 @@ let private mapNodeToTriviaNode (node: Node) =
     )
     
 let private commentIsAfterLastTriviaNode (triviaNodes: TriviaNode list) (range: range) =
-    triviaNodes
-    |> List.filter isMainNodeButNotAnonModule
-    |> List.forall (fun tn -> tn.Range.EndLine < range.StartLine)
+    match List.filter isMainNodeButNotAnonModule triviaNodes with
+    | [{ Type = MainNode("SynModuleOrNamespaceSig.NamedModule") }] -> true
```

Comment:
I needed this for comments in an empty module in a signature file, which has the named module being matched as a trivia node but which end line is equal to or greater than the comment

Reply:
I checked this, and I don't see any other way around this.
Looks good.